src/style.css

```diff
@@ -1 +1,73 @@
-/* add styles here */
+body {
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+  color: #3d4e61;
+}
+
+@font-face {
+  font-family: Avenir-Book;
+  font-style: normal;
+  font-weight: normal;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  font-style: bold;
```

Comment:
The 'font-style' property doesn't accept 'bold' as a value. You should remove this line.